.travis.yml

```diff
@@ -32,10 +32,8 @@ matrix:
       - php: 5.6
         env: SYMFONY_VERSION='~2.3.0'
       - php: 5.6
-        env: SYMFONY_VERSION='~2.5.0'
-      - php: 5.6
         env: SYMFONY_VERSION='~2.7.0'
       - php: 5.6
         env: SYMFONY_VERSION='~2.8.0'
       - php: 5.6
-        env: SYMFONY_VERSION='~3.0.0'
+        env: SYMFONY_VERSION='~3.1.5'
```

Comment:
I would not include 3.1 at all. It is the latest stable, so it is covered by the normal tests already, and it will become irrelevant very fast after normal tests are bumping to 3.2 when it gets released. My own policy is to add only LTS version as extra jobs.